src/VisualStudio/Core/Def/Interactive/VsInteractiveWindowProvider.cs

```diff
@@ -125,6 +120,14 @@ void closeEventDelegate(object sender, EventArgs e)
             window.InitializeAsync();
 
             LogSession(LogMessage.Window, LogMessage.Create);
+
+            static string GetFrameworkForTitle(InteractiveHostPlatform platform) => platform switch
+            {
+                InteractiveHostPlatform.Desktop64 => " (.NET Framework " + ServicesVSResources.Bitness64 + ")",
```

Comment:
📝 This makes assumptions about the sentence structure of other languages, which may or may not hold